code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -437,6 +439,9 @@
 				var/obj/item/clothing/head/hat = H.head
 				H.sight |= hat.vision_flags
 
+				if(hat.darkness_view && hat.darkness_view < minimum_darkness_view) // Pick the lowest of the two darkness_views between the glasses and helmet.
```

Comment:
I do wonder if `hat.darkness_view < minimum_darkness_view` should be enclosed in separate parenthesis or not.